device/iothub-java-client/src/main/java/com/microsoft/azure/iothub/transport/amqps/AmqpsTransport.java

```diff
@@ -201,7 +212,8 @@ public void sendMessages() throws IOException, IllegalStateException
                     // Codes_SRS_AMQPSTRANSPORT_15_036: [The function shall create a new Proton message from the IoTHub message.]
```

Comment:
Please log that message has expired, sent, send successfull, receive along with their timestamps where ever appropriate in this file.